test/client/main.cpp

```diff
@@ -1019,13 +1009,16 @@ int main(int argc, char* argv[])
     }
 
     util::Optional<sync::Session::Config::ClientReset> client_reset_config;
+    // These options are no longer supported in the ClientReset config structure
+#if 0
```

Comment:
How are these changes related to logging?

Reply:
Removed code